piksi_tools/console/settings.yaml

```diff
@@ -1046,7 +1046,7 @@
   expert: false
   type: string
   units: N/A
-  default value: '23,29,65,72,74,81,117,134,136,144,163,165, 166,167,171,175,181,185,187,188,189,190, 257,258,259,520,522,524,526,527,528,1025, 2305,2306,30583,65280,65282,65535' 
+  default value: '23,65,72,74,81,97,117,134,136,137,138,139, 144,149,163,165,166,167,171,175,181, 185,187,188,189,190,257,258,259,520, 522,524,526,527,528,1025,2304,2305, 2306,30583,65280,65282,65535'
```

Comment:
@silverjam 2304 is currently excluded from UART1 due to bandwidth issues.

Reply:
fixed